auth_signup_verify_email/__openerp__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# © 2015 Antiun Ingeniería, S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    "name": "Verify email at signup",
+    "summary": "Force uninvited users to use a good email for signup",
+    "version": "8.0.1.0.0",
+    "category": "Authentication",
+    "website": "https://odoo-community.org/",
+    "author": "Antiun Ingeniería, S.L., Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "application": False,
+    "installable": True,
+    "external_dependencies": {
+        "python": [
+            "validate_email",
+        ],
+    },
+    "depends": [
+        "auth_signup",
```

Comment:
If you don't depend on the *website* module, then what is your reason to add this module here?